Formula/brigade-cli.rb

```diff
@@ -0,0 +1,34 @@
+class BrigadeCli < Formula
+  desc "Brigade command-line interface"
+  homepage "https://brigade.sh"
+  url "https://github.com/brigadecore/brigade.git",
+      tag:      "v2.0.0",
+      revision: "b2f0aa11555f785894059d35f2646ccc05ab52f0"
+  license "Apache-2.0"
+  head "https://github.com/brigadecore/brigade.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["SKIP_DOCKER"] = "true"
+    ENV["VERSION"] = "v#{version}"
+
+    system "make", "hack-build-cli"
+
+    os = `go env GOOS`.strip
+    arch = `go env GOARCH`.strip
+    bin.install "bin/brig-#{os}-#{arch}" => "brig"
+  end
+
+  test do
+    system bin/"brig", "init", "--id", "foo"
+    assert File.directory? testpath/".brigade"
+
+    version_output = shell_output(bin/"brig version 2>&1")
+    assert_match "Brigade client:", version_output
+    if build.stable?
+      commit = stable.instance_variable_get(:@resource).instance_variable_get(:@specs)[:revision][0..6]
+      assert_match "Brigade client: version v#{version} -- commit #{commit}", version_output
+    end
```

Comment:
```suggestion
    return unless stable?

    commit = stable.specs[:revision][0..6]
    assert_match "Brigade client: version v#{version} -- commit #{commit}", version_output
```
I'm not a fan of using `stable.specs` here -- I'm pretty sure that's a private API.